packages/zoe/src/contractSupport/bondingCurves.js

```diff
@@ -98,7 +98,7 @@ export const getOutputPrice = (
   const numerator = BigInt(outputValue) * BigInt(inputReserve) * BIG_10000;
   const denominator =
     (BigInt(outputReserve) - BigInt(outputValue)) * oneMinusFeeScaled;
-  return Nat(Number(numerator / denominator));
+  return Nat(Number(numerator / denominator)) + 1;
```

Comment:
should we use natSafeMath here instead of adding with `+`?

Reply:
We should do neither. We should do the addition within the BigInt world before converting back to the precision limited world.
```suggestion
  return Nat(Number(numerator / denominator + BigInt(1)));
```